### ISSUE TITLE
OnfidoActivity - IllegalStateException: Can not perform this action after onSaveInstanceState

### ISSUE THREAD

**Issue by xrigau** [onfido/onfido-android-sdk#213]
#### What was the expected behaviour?

The app shouldn't crash, the Onfido flow should work just fine when it's being launched.

#### What happened instead?

The app crashes as the Onfido SDK is launched.

#### Version info:

- `onfido-capture-sdk` version: `com.onfido.sdk.capture:onfido-capture-sdk-core:12.3.0` - Notice we're using `onfido-capture-sdk-core`
- Android API version(s): According to Firebase, this crash has occurred in Android 8, 11 and 12.

- `compileSdkVersion`: 33
- `targetSdkVersion`: 31
- `minSdkVersion`: 27
- Support library version (if applicable): N/A

#### Steps to reproduce:

We don't know, we haven't been able to reproduce this issue.

#### Onfido Flow initialization code:

```kotlin
val tokenExpirationHandler = // ...
    val config = OnfidoConfig.builder(context)
        .withSDKToken("token", tokenExpirationHandler)
        .build()
    val intent = onfido.createIntent(config)
    startActivity(intent)
```

#### Issue severity information (number of users/crashes if applicable):

This issue has 24 crash events affecting 21 users for the past 2 weeks.

#### Additional LogCat output:

Stacktrace from Firebase:

```
Fatal Exception: java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
       at androidx.fragment.app.FragmentManager.checkStateLoss(FragmentManager.java:1601)
       at androidx.fragment.app.FragmentManager.ensureExecReady(FragmentManager.java:1702)
       at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1745)
       at androidx.fragment.app.FragmentManager.popBackStackImmediate(FragmentManager.java:845)
       at androidx.fragment.app.FragmentManager.popBackStackImmediate(FragmentManager.java:796)
       at com.onfido.android.sdk.capture.ui.OnfidoActivity.setFragment$lambda-10(OnfidoActivity.java:1)
       at com.onfido.android.sdk.capture.ui.OnfidoActivity.$r8$lambda$hHOSSl5a3iOHp7TkDJoJgXj4158(OnfidoActivity.java)
       at androidx.camera.camera2.internal.Camera2CameraImpl$$InternalSyntheticLambda$4$eed53b49a6a694889cf7ff43e02a590c50888ae8ba6790d439b82cd75c62da75$1.run$bridge(Camera2CameraImpl.java:44)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:223)
       at android.app.ActivityThread.main(ActivityThread.java:7945)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:603)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

#### Other useful info:

According to Firebase this occurs when the app is in the background, which makes me think this has to do with the user leaving the app as the Onfido SDK is being launched and then resuming the app again, or something like that - but not 100% sure.

![image](https://user-images.githubusercontent.com/1626673/194596817-c31b8491-3014-4c09-a0b8-e28fbb13c1a3.png)

Does not appear to be OS version specific or device specific.

**Comment by Tobi0077** [comment]
Hi @xrigau 

We came across this issue in 12.3.1 which was already discussed with our engineering team which mentioned that this is a non-fatal meaning this is being handled already by the SDK, but we are still logging them.

**Comment by xrigau** [comment]
That doesn't seem right, if this was a non-fatal error we wouldn't see the report in our Firebase Crashlytics dashboard, unless I'm missing something?

**Comment by Tobi0077** [comment]
Hi @xrigau 

Apologies for the delay in response. The issue has been fixed on our latest release [Release 13.0.1 · onfido/onfido-android-sdk](https://github.com/onfido/onfido-android-sdk/releases/tag/13.0.1). 

Apologies once more for the inconvenience.

**Comment by xrigau** [comment]
Awesome, thanks for the update @Tobi0077 !
